synapse/lib/storm.lark

```diff
@@ -48,13 +48,17 @@ vareval: _varvalu
 valuvar: _varname _WS? "=" _WS? _valu
 
 // We prioritize because 'for' looks like a command
-forloop.1: "for" _WS? (_varname | varlist) _WS? "in" _WS? _varvalu _WS? subquery
+forloop.1: "for" _WS (_varname | varlist) _WS? "in" _WS? _varvalu _WS? subquery
```

Comment:
do we still need these things prioritized if they are no longer parsable as commands via reserved words?

Reply:
We do not, but it doesn't hurt anything either. I'll catch that on the next PR.